channeldb/channel.go

```diff
@@ -403,18 +403,30 @@ var (
 
 	// ChanStatusCoopBroadcasted indicates that a cooperative close for this
 	// channel has been broadcasted.
+	// Deprecated: use ChanStatusLocalInitCoopBroadcast or
```

Comment:
just a drive by comment, so ignore if this has already been discussed: can't  we add this info (who initiated) under a new key in the db? So we wouldn't have to change the meaning and handling of this flag, and only check this key for the value, if not fall back to a default.

Reply:
if adding a new key to the db is preferable to using up 2 more bits in channel status then this would definitely be the way to go. Would you say only set this new field for coop? Since we already know the initiator from the close tx in other scenarios.